CHANGELOG.md

```diff
@@ -8,7 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Rename `IdsGenerator` to `IdGenerator`
-  ([#350])(https://github.com/open-telemetry/opentelemetry-python/pull/350)
+  ([#350](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/350))
+- Rename `DefaultSpan` to `NonRecordingSpan`
```

Comment:
Since this change is in the API only and isn't user facing for any users of the packages of this repo, I would skip the changelog